src/app/src/App.js

```diff
@@ -1,53 +1,16 @@
 import React, { Component } from 'react';
-import { number } from 'prop-types';
-import { connect } from 'react-redux';
 
-import logo from './logo.svg';
+import Map from './Map';
```

Comment:
Nitpick but consider giving this a more specific name to distinguish it from the builtin JS `Map` datatype and the `map` operation.